test/extended-priv/testdata/files/rpm-ostree-fake-install-noop.sh

```
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# Fake rpm-ostree that silently succeeds on install/override commands.
+# All other commands are forwarded to the real rpm-ostree backup.
+if [[ "$@" == *"install"* ]] || [[ "$@" == *"override"* ]]; then
```

Shellcheck (0.11.0)

[error] Arrays implicitly concatenate in [[ ]]. Use a loop (or explicit * instead of @).

(SC2199)